elements/link.html

```diff
@@ -25,14 +25,14 @@
     </div>
     <div class="how-to-remember-box how-to-remember-english">
       <h4>How to Remember</h4>
-      <p>While you can’t read a book by it’s cover, you can get a pretty good idea what it’s about by the title.  A <code>&lt;title></code> is a quick way of identifying what the document is about.</p>
+      <p>A chain is made with links, which attaches one item to another. </p>
     </div>
   </div>
   <pre class="code-example"><code>
-      &lt;html>
-        &lt;head>
-          <strong>&lt;title></strong>Document title<strong>&lt;/title></strong>
-        &lt;/head>
-      &lt;/html>
+      &lt;html&gt;
+        &lt;head&gt;
+          <strong>&lt;link</strong> rel="stylesheet" href="style.css">
```

Comment:
what's the style issue?
and yes  it should be      <strong>&lt;link</strong> rel="stylesheet"
href="style.css"&gt;




[image: photo]
*Erica Naughton*
Designer/Developer, Adorable.io
+1 866 440 4593 | http://adorable.io | 123 E. Main St - 2nd Floor, Madison
WI 53715
<http://twitter.com/adorableio>  <http://facebook.com/adorableio>
✔ Get a Signature like this: Click Here!
<http://ws-promos.appspot.com/r?rdata=eyJydXJsIjogImh0dHA6Ly93d3cud2lzZXN0YW1wLmNvbS9lbWFpbC1pbnN0YWxsP3dzX25jaWQ9NjcyMjk0MTIwJnV0bV9zb3VyY2U9ZXh0ZW5zaW9uJnV0bV9tZWRpdW09ZW1haWwmdXRtX2NhbXBhaWduPXByb21vXzU3MTM1NzMyNTA1OTY4NjQiLCAiZSI6ICI1NzEzNTczMjUwNTk2ODY0In0=>

On Wed, Nov 4, 2015 at 5:26 PM jremsik-gpsw <notifications@github.com>
wrote:

> In elements/link.html
> <https://github.com/LMHT/LMHT/pull/15#discussion_r43956539>:
>
> >      </div>
> >    </div>
> >    <pre class="code-example"><code>
> > -      &lt;html>
> > -        &lt;head>
> > -          <strong>&lt;title></strong>Document title<strong>&lt;/title></strong>
> > -        &lt;/head>
> > -      &lt;/html>
> > +      &lt;html&gt;
> > +        &lt;head&gt;
> > +          <strong>&lt;link</strong> rel="stylesheet" href="style.css">
>
> No, and this is a style issue. The &gt; is the > at the end of the line.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/LMHT/LMHT/pull/15/files#r43956539>.
>